BrainPortal/app/views/tasks/show.html.erb

```diff
@@ -61,29 +61,20 @@
 
     <!-- FAILURE RECOVERY -->
     <% if CbrainTask::FAILED_STATUS.include?(@task.status) %>
-      <%
-        label = (
-          case @task.status
-          when /Setup/       then "Retry Setting Up"
-          when /Cluster/     then "Retry Processing"
-          when /PostProcess/ then "Retry Post Processing"
-          end
-        )
-      %>
-      <%= link_to label, { :action => 'operation', :operation => 'recover', 'tasklist[]' => @task.id }, :class  => "button menu_button", :method => :post %>
+      <%= link_to 'Retry Failed', { :action => 'operation', :operation => 'recover', 'tasklist[]' => @task.id }, :class  => "button menu_button", :method => :post %>
```

Comment:
Right. Let me document more, sorry. As can be seen in the original code (introduced in commit b5297ced in April 2010 in https://github.com/aces/cbrain/commit/b5297ced#diff-c7aa0f93adfa9b6ebe3258e64f513f13R52 ), the test in the view code was

```ruby
if @task.status =~ /^Failed*/
```

This was changed to use CbrainTask::FAILED_STATUS this past November.

Since that list CbrainTask::FAILED_STATUS contains too many statuses, I suggest we create CbrainTask::RECOVERABLE_STATUS with only the good six status keywords (and we can redefined FAILED_STATUS to be that list plus 'Terminated').